### ISSUE TITLE
PPO policy collapses to deterministic within ~30 episodes

### ISSUE THREAD

**Issue by clankur** [clankur/doomer#1]
## Problem

During smoke testing of the PPO implementation, the policy entropy collapses to 0.0 by ~episode 30, meaning the agent becomes fully deterministic very early in training. After collapse, the running reward degrades (from -228 back to -274 in a 50-episode run) because the agent can no longer explore.

## Observed behavior

```
[Episode    25] entropy=0.5039
[Episode    30] entropy=0.0457
[Episode    35] entropy=0.0000   <-- fully deterministic from here
```

## Possible causes

- `entropy_coeff=0.01` may be too low to counteract the gradient signal pushing toward a peaked distribution
- Value loss (~2000-4000) dominates total loss early on, drowning out the entropy bonus
- The shared backbone may propagate large value gradients into the policy head

## Potential fixes

- Increase `entropy_coeff` (try 0.05 or 0.1)
- Reduce `value_loss_coeff` to prevent value loss from dominating
- Add gradient clipping (`max_grad_norm`)
- Separate learning rates or gradient scaling for policy vs value heads

## Reproduction

```bash
uv run python -m train --config-name=local_test ++paths.model_name=ppo_smoke_000
```

Watch the `entropy` column in the logs.
